frontend/src/features/permits/components/dashboard/ApplicationStepPage.tsx

```diff
@@ -50,7 +54,7 @@ export const ApplicationStepPage = ({
     setApplicationData,
     shouldEnableQuery,
     isInvalidRoute,
-  } = useApplicationDetailsQuery(applicationStep, permitId, permitType);
+  } = useApplicationDetailsQuery({ applicationStep, permitId, permitType });
```

Comment:
My thought here was that since there 3 of the 4 parameters for useApplicationDetailsQuery are optional, an object makes passing arguments more explicit therefore avoiding confusion, especially in circumstances when we are not providing all of the possible arguments that useApplicationDetailsQuery accepts.